Dockerfile

```diff
@@ -25,6 +25,9 @@ RUN sh /home/modules/install_cmake.sh
 COPY install_ninja.sh /home/modules/install_ninja.sh
 RUN sh /home/modules/install_ninja.sh
 
+COPY install_clang.sh /home/modules/install_clang.sh
```

Comment:
As mentioned in #3, I think this should be before `cmake` and `ninja` so that bumping the "small" projects doesn't require rebuilding clang. We can install a system-wide `cmake` and `ninja-build` for that. We should probably also just modify `PATH` using `ENV` commands rather than a wrapper script so that it's always available.

Reply:
OK. Reordered.